src/spikeinterface/core/base.py

```diff
@@ -470,7 +471,32 @@ def clone(self) -> "BaseExtractor":
         return clone
 
     def check_if_dumpable(self):
-        return _check_if_dumpable(self.to_dict())
+        # Find all nested _is_dumpable
+        dict_repr = self.to_dict()
+        kwargs = dict_repr["kwargs"]
```

Comment:
why not kwargs = self._kwargs to avoid the to_dict ?

Reply:
good point, done!